specs/Deployment/DeploymentAPI.md

```diff
@@ -312,6 +502,18 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         [contract(DeploymentContract, 2)]
         [overload("Initialize")]
         static DeploymentResult Initialize(Microsoft.Windows.ApplicationModel.WindowsAppRuntime.DeploymentInitializeOptions deploymentInitializeOptions);
+
+        /// Checks the status of the WindowsAppRuntime of the current package and attempts to
+        /// repair already installed WinAppSDK packages.
+        [contract(DeploymentContract, 3)]
+        static DeploymentResult Repair();
+
+        /// Checks the status of the WindowsAppRuntime of the current package and attempts to
+        /// repair already installed WinAppSDK packages, while applying the DeploymentInitializeOptions
```

Comment:
Comment here incorrectly refers to DeploymentInitializeOptions rather than DeploymentRepairOptions